pkg/apis/experimental/types.go

```diff
@@ -461,3 +461,75 @@ type JobCondition struct {
 	// Human readable message indicating details about last transition.
 	Message string `json:"message,omitempty"`
 }
+
+// IngressPoint encapsulates the inputs needed to connect an alias to endpoints.
+// It represents a rule collection of inbound connections from the external network
+// that would be satisfied by a load balancer.
+type IngressPoint struct {
```

Comment:
+1

On Sep 15, 2015, at 9:17 PM, Tim Hockin <notifications@github.com> wrote:

In pkg/apis/experimental/types.go
<https://github.com/kubernetes/kubernetes/pull/13947#discussion_r39585412>:

> @@ -461,3 +461,75 @@ type JobCondition struct {
>  	// Human readable message indicating details about last transition.
>  	Message string `json:"message,omitempty"`
>  }
> +
> +// IngressPoint encapsulates the inputs needed to connect an alias to endpoints.
> +// It represents a rule collection of inbound connections from the external network
> +// that would be satisfied by a load balancer.
> +type IngressPoint struct {

can I argue for just Ingress ?

—
Reply to this email directly or view it on GitHub
<https://github.com/kubernetes/kubernetes/pull/13947/files#r39585412>.